language/move-lang/src/lib.rs

```diff
@@ -29,9 +29,12 @@ use shared::Address;
 use std::{
     collections::HashMap,
     fs::File,
-    io::{self, Read, Write},
+    io::{self, ErrorKind, Read, Write},
+    path::Path,
 };
 
+const MOVE_EXTENSION: &str = "move";
```

Comment:
Fine for now, but I do have this extension listed in a test util somewhere. Should try to get it down to a single constant at some point

Reply:
Agreed